src/components/alert.tsx

```diff
@@ -0,0 +1,79 @@
+interface AlertProps {
+  status?: string
+  hidden?: boolean
+}
+
+export function Alert({ status, hidden }: AlertProps) {
+
+  let title: string = '';
+  let message: string = '';
+
+  switch (status) {
+    case 'error':
+      title = 'Error';
+      message = 'An error occurred while uploading your video. Please check the CLI logs for more info.';
+      break;
+    case 'sourced':
+      title = 'Video is not processing';
+      message = 'Make sure to run next-video sync. The currently loaded video is the source file.';
+      break;
+    default:
+      title = 'Upload in progress...';
+      message = 'Your video file is being uploaded. The currently loaded video is the source file.';
+      break;
+  }
+
+  return (
+    <>
+      <style>{
+        /* css */`
+        .alert {
+          position: absolute;
+          inset: 1em;
+          bottom: auto;
+          padding: .75rem 1rem;
+          border-radius: 1rem;
+          color: hsl(0, 0%, 100%);
+          background-color: hsl(240 10% 3.9% / .7);
+          border: 1px solid hsl(240 3.7% 15.9%);
+          transition: visibility 0s, opacity .25s;
+          visibility: visible;
+          opacity: 1;
+        }
+
+        .alert[hidden] {
+          display: block;
+          transition: visibility 1s, opacity 1s;
+          visibility: hidden;
+          opacity: 0;
+        }
+
+        .alert svg {
+          position: absolute;
+        }
+
+        .alert h5 {
+          line-height: 1;
+          font-weight: 500;
+          margin-bottom: 0.25rem;
+          padding-left: 1.75rem;
+          font-size: inherit;
+        }
+
+        .alert div {
+          padding-left: 1.75rem;
+          font-size: 0.875rem;
+          line-height: 1.25rem;
+        }
+        `
+      }</style>
```

Comment:
Is there some Next magic going on here that I don't know about? Technically this goes against the HTML spec and these look like they're just unscoped CSS classes.